docs/modules/tile-converter/cli-reference/tile-converter.md

```diff
@@ -76,9 +78,9 @@ NodeJS 14 or higher is required.
-| egm                       | \*                        | \*                        | location of the Earth Gravity Model (\*.pgm) file to convert heights from ellipsoidal to gravity-related format or "None" to not use it, default: "./deps/egm2008-5.pgm". A model file can be loaded from GeographicLib https://geographiclib.sourceforge.io/html/geoid.html                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| slpk                      | \*                        |                           | (Deprecated since version 4.3.0) <br> Whether the converter generates \*.slpk (Scene Layer Package) I3S output files. Note: For versions 4.3.0 and up \*.slpk is the default output wihtout this option specified.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| 7zExe                     | \*                        |                           | (Deprecated since version 4.1.0) <br> location of 7z.exe archiver to create slpk on Windows OS, default: "C:\\Program Files\\7-Zip\\7z.exe". Note: For versions 4.1.0 and up 7z is not needed and tile-converter uses internal zip module instead.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
This option is removed. Remove it from the docs

Reply:
Is it removed for 4.0.x and below? I thought maybe old versions need this options

Reply:
Added as "(Removed since version 4.1.0)"